Makefile

```diff
@@ -28,11 +28,10 @@ install: certs
 uninstall:
 	kubectl config set-context --current --namespace $(NAMESPACE)
 	helm uninstall connaisseur
-	kubectl delete ns $(NAMESPACE) --timeout=120s
```

Comment:
I found it to be impratical when using predefined secrets, like image pull secrets or general authentication secrets. They would be deleted as well, which might not be the wished for behavior.

Reply:
Good point. Would add that part to https://github.com/sse-secure-systems/connaisseur/blob/master/setup/README.md#cleanup then. "We do not delete the namespace, since you might have deployed things in there, which we would inadvertently delete. If you want to get rid of the namespace, run `kubectl delete ns WHATEVEROURDEFAULTIS`"

Reply:
resolved in 3b7d64cd6e6c5c09c2f7370d57ddb8a3b6c8869c